java/arcs/core/data/expression/Evaluator.kt

```diff
@@ -44,11 +44,13 @@ class ExpressionEvaluator(
   }
 
   override fun <T> visit(expr: Expression.FieldExpression<T>, ctx: Scope): Any? =
-    (if (expr.qualifier == null) {
-      ctx
-    } else {
-      expr.qualifier.accept(this, ctx) as Scope?
-    }).apply {
+    (
+      if (expr.qualifier == null) {
+        ctx
+      } else {
+        expr.qualifier.accept(this, ctx) as Scope?
+      }
+      ).apply {
```

Comment:
another case of having a complex multi-line expression fubaring if used as a method qualifier. 

This snippet will be fixed in an updating paxel PR, so ok to leave as is